apps/style/code-studio/pd.scss

```diff
@@ -34,6 +34,14 @@
     white-space: normal;
     font-weight: normal;
   }
+
```

Comment:
I'm using the bootstrap 3.0 styles, so yes. I'll move the other styles into the container.